lib/ui/myplot/PlotDetail.dart

```diff
@@ -203,7 +211,21 @@ class _PlotDetailState extends State<PlotDetail> {
   }
 
   void _renameAction(PlotDetailViewModel viewModel) {
-    viewModel.rename("test"); //TODO: add the UI for input when ready
+    InputAlert.present(_renameInputAlert(viewModel), context);
+  }
+
+  InputAlert _renameInputAlert(PlotDetailViewModel viewModel) {
+    return InputAlert(
+      viewModel: InputAlertViewModel(
+        cancelActionText: _LocalisedStrings.cancelAction(),
+        confirmActionText: _LocalisedStrings.confirm(),
+        titleText: _LocalisedStrings.renameAction(),
+        hint: _LocalisedStrings.cropName(),
```

Comment:
Not sure if I would name this string as `cropName` or `hintCropName` , what do you think?

Reply:
cropName is fine, it's a LocalisedString and we name it for what it is. It is "Crop name" not "Hint crop name" 😄